packages/viewer/src/scenes/scene.ts

```diff
@@ -55,6 +55,15 @@ export class SceneItemOperationsExecutor
     );
   }
 
+  public clear(): SceneItemOperationsExecutor {
```

Comment:
`clear` is pretty vague about what's being cleared. How about `clearMaterialHighlight()`?